packages/tokamak/contracts-bedrock/package.json

```diff
@@ -57,10 +57,13 @@
     "lint": "pnpm lint:fix && pnpm lint:check"
   },
   "devDependencies": {
+    "@eth-optimism/contracts-bedrock": "workspace:*",
     "@typescript-eslint/eslint-plugin": "^6.21.0",
     "@typescript-eslint/parser": "^6.21.0",
     "tsx": "^4.7.0",
-    "typescript": "^5.4.5",
-    "@eth-optimism/contracts-bedrock": "workspace:*"
+    "typescript": "^5.4.5"
+  },
+  "dependencies": {
+    "@openzeppelin/contracts": "^5.2.0"
```

Comment:
Why are you importing `@openzeppelin/contracts` v5.2.0? The git submodule in this package already contains `openzepplin/contracts` v5.0.1.